packages/apps/spaces/app/organization/[id]/src/components/Tabs/panels/AboutPanel/branches/Branches.tsx

```diff
@@ -27,10 +27,15 @@ import {
 
 interface BranchesProps {
   id: string;
+  isReadOnly?: boolean;
   branches?: Organization['subsidiaries'];
 }
 
-export const Branches: React.FC<BranchesProps> = ({ id, branches = [] }) => {
+export const Branches: React.FC<BranchesProps> = ({
+  id,
+  isReadOnly,
```

Comment:
can we rename this to visible? read only would have a different sense in terms of displaying

Reply:
nope. the prop is actually doing the fields read-only not hiding them.

Reply:
you are hiding the button below based on it. the readonly is don in the select component